case-study-hydropro-solutions.mdx

```diff
@@ -0,0 +1,72 @@
+tags: ['case-study', 'community']
+published: true
+slug: case-study-hydropro-solutions
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/sava-labs-case-study-og@2x.png
+---
+
+<ArticleImage
+  src="https://us-east-1.storage.xata.sh/uq2d573lf36fvlvl6td0s0i4spj3kohc"
+  width="800"
+  height="405"
+  alt="Savas labs hero image"
+  position="center"
+/>
+
+- **Company:** HydroPro Solutions
+- **Industry:** Water utilities
+- **Location:** Texas & California
+- **Size:** 30 employees
+
+[HydroPro Solutions](https://hydroprosolutions.com/) is a leading sales and distribution company providing smart water meters and sewage monitoring systems to small and medium-sized water utilities across Texas and California. With a team of 30 employees, they specialize in advanced water infrastructure services, including pipeline pressure monitoring and cyber threat detection, ensuring municipalities can manage their water resources effectively and securely.
+
+## Challenges and the task ahead
+
+As the sole platform engineer at HydroPro Solutions, Chris Hiester faced a significant challenge: building an entire software platform to manage large-scale water meter replacement projects. These projects involved managing data for up to 20,000 water meters in a single city, including customer information, meter readings, work orders, and high-resolution images of the installations.
+
+Chris needed a database solution that could handle vast amounts of relational data while being easy to manage as a single developer. The system also needed to support search capabilities for quickly locating customer records and photos, avoiding costly billing disputes. Additionally, HydroPro needed the platform to integrate smoothly into their existing tech stack, which included Next.js, Vercel, and Google Cloud Vision.
+
+> We needed a PostgreSQL system that would just work and perform well without the hassle of dealing with core database infrastructure. As a single engineer, I didn’t want to spend time managing configurations—Xata’s APIs and search capabilities made it a perfect fit.\
+> _Chris Hiester, Lead Application Engineer at HydroPro Solutions_
+
+## Solution
+
+After evaluating several database solutions, Chris chose Xata for its powerful Postgres-based relational database features and ease of use. Xata provided a developer-friendly Postgres platform with robust search functionality and the ability to manage large datasets efficiently.
+
+With Xata, HydroPro was able to:
+
+- **Store and manage vast amounts of relational data:** Including customer records, water meter data, and work orders.
+- **Use advanced search capabilities:** Enabling quick and accurate data retrieval for customer service, ensuring no billing errors or discrepancies during meter installations.
+- **Manage file attachments:** Storing photos taken during meter installations and using Google Cloud Vision to read meter serial numbers, which are automatically linked to customer records.
```

Comment:
Not the best wording. I'm trying to sneak in that they used image transformations. Any better ideas?

```suggestion
- **Manage file attachments:** Storing photos taken during meter installations and using Google Cloud Vision to read meter serial numbers, which are automatically linked to customer records. Image transformations are then used when delivered in their mobile and web apps.
```

Reply:
Gotcha. I have tweaked so it reads a little less confusing for people reading this for the first time.

"Manage file attachments:** Storing photos taken during meter installations and using Google Cloud Vision to read meter serial numbers, which are automatically linked to customer records. Xata's image transformation capabilities create smaller thumbnails of the images, allowing field teams using mobile devices to quickly view and manage high-resolution photos in real-time, improving on-site efficiency."